test/inductor/test_fused_attention.py

```diff
@@ -118,14 +118,16 @@ def dot_prod_attention(
             )
 
         for dtype in [torch.float, torch.half]:
-            if self.device == "cpu" and dtype == torch.half:
-                continue
+            atol = 0.001
             rtol = 1.3e-6 if dtype == torch.float else 0.7
-            self._check_common(dot_prod_attention, dtype=dtype, atol=0.001, rtol=rtol)
+            if self.device == "cpu" and dtype == torch.half:
+                atol = 2e-3
+                rtol = 1e-2
```

Comment:
Please leave a comment why tolerance needs to be raised for CPU, but not for CUDA

Reply:
Sorry for the late reply, as I am currently on holiday. `rtol` on CPU does not need a value as large as 0.7.